src/middlewared/middlewared/plugins/smb_/util_groupmap.py

```diff
@@ -181,7 +181,7 @@ def delete_groupmap_entry(
         hdl.delete(tdb_key)
 
 
-def list_foreign_group_memberships(
+def foreign_group_memberships(
```

Comment:
Docs would be nice.

Reply:
Killed that function off since it was redundant to what was already provided via `query` API.